packages/claw/package.py

```diff
@@ -12,42 +12,42 @@ class Claw(CMakePackage):
        generates architecture specific code decorated with OpenMP or OpenACC"""
 
     homepage = 'https://claw-project.github.io/'
-    git      = 'https://github.com/claw-project/claw-compiler.git'
-    maintainers = ['clementval']
+    git = 'https://github.com/claw-project/claw-compiler.git'
+    maintainers = ['FrostyMike']
 
     version('master', branch='master', submodules=True)
+    version('2.1', tag='v2.1', submodules=True)
```

Comment:
@elsagermann or @FrostyMike is it equivalent to use commit or tag (the one below use commits)

Reply:
It is equivalent if I am not wrong, tag being also a commit hash

Reply:
Commit is permanent, while tag can be moved to reference another commit.

Reply:
Does the tag commit changes a lot though? I see your point though, you could change it for the other tag then ?

Reply:
This one might change if bugs are found (there was little testing with cosmo & icon so far).

Reply:
ok so we go for tag

Reply:
nonetheless if we need to fix, we'll just do a new claw release. So the association should not change.